package.json

```
@@ -171,7 +171,7 @@
           },
           "calva.enableClojureLspOnStart": {
             "type": "boolean",
-            "markdownDescription": "**Recommended**: Start [clojure-lsp](https://calva.io/clojure-lsp/) automatically when Calva starts",
+            "markdownDescription": "**Recommended**: Start [clojure-lsp](https://calva.io/clojure-lsp/) automatically when Clojure files are opened",
```

```suggestion
            "markdownDescription": "**Recommended**: Start [clojure-lsp](https://calva.io/clojure-lsp/) automatically when a Clojure file in a project is opened",
```

Aside from the suggested change to the description above, I think this setting should be renamed to `calva.startClojureLspOnClojureFileOpened`, so the name is more descriptive of what it actually does.

---

Resolved in 7234496a